qiime/split_libraries_lea_seq.py

```diff
@@ -194,6 +201,8 @@ def get_LEA_seq_consensus_seqs(fwd_read_f, rev_read_f,
                                  primer_mismatch_count,
                                  seq_too_short_count,
                                  total_seqs_kept)
+    if consensus_seq_lookup is None:
```

Comment:
I don't think the return value of `get_consensus_seqs_lookup` will ever be `None`, so I don't think this statement will ever evaluate to `True`. Can you please fix and add a unit test for this case?